rpcd/playbooks/roles/elasticsearch/tasks/elasticsearch_install.yml

```diff
@@ -31,7 +31,9 @@
   pip:
     name: "{{ item }}"
     state: latest
-    extra_args: "{{ pip_install_options | default('') }}"
+    extra_args: >-
+      {{ (pip_install_upper_constraints is defined) | ternary('--constraint ' + pip_install_upper_constraints | default(''),'') }}
```

Comment:
The upper constraints are never defined for the rpcd/playbooks, but this should be OK as the containers should be configured to be restricted by the pip install role. If you'd like to be extra safe, add https://github.com/openstack/openstack-ansible/blob/stable/ocata/playbooks/inventory/group_vars/all.yml#L64 to the rpcd group_vars/all.yml